pandas/core/generic.py

```diff
@@ -1354,11 +1356,15 @@ def to_pickle(self, path, compression='infer'):
             File path
         compression : {'infer', 'gzip', 'bz2', 'xz', None}, default 'infer'
             a string representing the compression to use in the output file
+        protocol : int
+            Int which indicates which protocol should be used by the pickler,
+            default HIGHEST_PROTOCOL (Pickle module).
```

Comment:
version added tag (the compression one goes above this), your tag should read 0.21.0